packages/manager/modules/pci/src/projects/index.js

```diff
@@ -1,19 +1,17 @@
 import angular from 'angular';
 import '@uirouter/angularjs';
-import 'oclazyload';
 
 const moduleName = 'ovhManagerPciProjectsLazyLoading';
 
-angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+angular.module(moduleName, ['ui.router']).config(
   /* @ngInject */ ($stateProvider) => {
     $stateProvider.state('pci.projects.**', {
       url: '/projects',
       lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
-
-        return import('./projects.module').then((mod) =>
-          $ocLazyLoad.inject(mod.default || mod),
-        );
+        return import('./projects.module').then((mod) => {
```

Comment:
you are still using ocLazyLoad so it should be kept in module dependencies

Reply:
true, fixed.